keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -226,16 +226,16 @@ def adjust_channels(x, divisor=8, min_value=None):
     Args:
         x: integer, input value.
         divisor: integer, the value by which a channel number should be
-            divisible, defaults to 8.
-        min_value: float, optional minimum value for the new tensor. If None,
-            defaults to value of divisor.
+            divisible. Defaults to `8`.
+        min_value: float, optional minimum value for the new tensor. When None,
+            is `divisor`. Defaults to `None`.
 
     Returns:
         the updated input scalar.
     """
 
     if min_value is None:
-        min_value = divisor
+        min_value = float(divisor)
```

Comment:
Do we need this change? Seems like the only non-docstring change so far so if possible it might be best to exclude it to keep this PR as safe as possible

Reply:
this is the change I was referring to to try rolling back from mobilenet. This is a non-docs change which could potentially be causing the breakage.